README.md

```diff
@@ -5,7 +5,7 @@ This repository contains all of the code necessary to run and test the Ably exam
 Click on any of the links below to find out more specific information on each example:
 
 - [Avatar Stack](/src/components/AvatarStack/README.md)
-- [Message Interactions](/src/components/MessageReactions/README.md)
+- [Message Interactions](/src/components/EmojiReactions/README.md)
```

Comment:
Message Reactions
```suggestion
- [Message Reactions](/src/components/EmojiReactions/README.md)
```

Reply:
That should be Emoji Reactions
```suggestion
- [Emoji Reactions](/src/components/EmojiReactions/README.md)
```

Sorry, that should be emoji reactions

Reply:
@ginotoro I updated this to be `Emoji Reactions`

Reply:
Great. Can you review and merge the changes?